AwsEncryptionSDK/runtimes/rust/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "aws-esdk"
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.80.0"
+keywords = ["cryptography", "security", "dynamodb", "ddb", "encryption", "client-side"]
```

Comment:
It seems we are limited to five keywords.

Reply:
Thanks